### DISCUSSION TITLE
Sleep Timer broken

### DISCUSSION THREAD

**Discussion by theAkito** [PaulWoitaschek/Voice#2281] [General]
I'm on app version 8.2.0, after the last update, and now the sleep timer is completely broken. It shows some times to pick which I don't need and even worse, I cannot select an exact time, even when I input a custom time. It only works in 5 minute steps, but I usually have specific timers like 7 or 12 or 17 minutes... Now I cannot use this anymore and this breaks my entire workflow.

This breaks the app for me and I need to downgrade ASAP.

(I am not able to open an issue right now, but since this is so urgent, I decided to still open this discussion rather than nothing, at all.)

**Comment by PaulWoitaschek** [comment]
Can you describe how this effects your workflow and what you workflow is that you need 7 minutes?
